src/op_analytics/datapipeline/etl/blockbatchloaddaily/ddl/blockbatch_daily/wallet_segments/daily_tx_activity_segments_v1__INSERT.sql

```diff
@@ -0,0 +1,175 @@
+WITH
+DailyWalletActivity AS (
+
+  SELECT
+    dt, chain, chain_id, network, from_address,
+    count_transactions, count_success_transactions, count_distinct_blocks,
+    count_distinct_success_blocks, count_distinct_active_hours,
+    count_distinct_to_address, count_distinct_success_to_address,
+    count_distinct_method_id, active_block_range, active_nonce_range,
+    active_time_range, sum_l2_gas_used, sum_success_l2_gas_used,
+    sum_l1_gas_used_unified, sum_success_l1_gas_used_unified, sum_tx_fee_native,
+    sum_success_tx_fee_native, sum_l2_fee_native, sum_success_l2_fee_native,
+    sum_l2_base_fee_native, sum_success_l2_base_fee_native, sum_l2_priority_fee_native,
+    sum_success_l2_priority_fee_native, sum_l2_legacy_extra_fee_native,
+    sum_success_l2_legacy_extra_fee_native, sum_l1_fee_native, sum_success_l1_fee_native,
+    sum_l1_base_fee_native, sum_success_l1_base_fee_native, sum_l1_blob_fee_native,
+    sum_success_l1_blob_fee_native, avg_l2_gas_price_gwei, avg_l2_priority_gas_price_gwei,
+    avg_l2_legacy_extra_gas_price_gwei, avg_l1_base_gas_price_gwei,
+    avg_l1_blob_base_gas_price_gwei, sum_input_zero_bytes, sum_success_input_zero_bytes,
+    sum_input_bytes_length, sum_success_input_bytes_length, sum_input_nonzero_bytes,
+    sum_success_input_nonzero_bytes, sum_estimated_size, sum_success_estimated_size
+  FROM INPUT_CLICKHOUSE('blockbatch_daily/aggtxs/daily_address_summary_v1')
+  WHERE
+    1=1
+    AND count_transactions > 0
+),
+
+DailyWalletCohorts AS (
+  -- Assigns segments and calculates per-wallet rates/averages
+  SELECT
+    dwa.*, -- Select all columns from DailyWalletActivity
+    -- Define segments based on daily transaction count
+    multiIf(
+      dwa.count_transactions >= 501, '>500_tx_per_day',    -- Tier5: Automated (High Freq)
```

Comment:
Maybe cut the tier definitions here, so that we're not locked in to them. 50+ doesn't necessarily mean automated, <15 doesn't necessarily mean organic